IOPool/Input/test/PoolGUIDTest_cfg.py

```diff
@@ -0,0 +1,31 @@
+# Configuration file for PoolInputTest
+
+import FWCore.ParameterSet.Config as cms
+import sys
+
+argv = []
+foundpy = False
+for a in sys.argv:
+    if foundpy:
+        argv.append(a)
+    if ".py" in a:
+        foundpy = True
+
+process = cms.Process("TESTRECO")
+process.load("FWCore.Framework.test.cmsExceptionsFatal_cff")
+
+process.maxEvents = cms.untracked.PSet(
```

Comment:
This can just be `process.maxEvents.input = -1`

Reply:
Right, it was easier to copy-paste than think about it. Replaced it now (will drop from the backports).